pubsub: panic if Ack is called twice on one message (#818)
google/go-cloud: internal/docs/pubsub/design.md

	// sub is the Subscription this message was received from.
	sub *Subscription

> 	// isAcked tells whether the message has already had its Ack method called.

Reviewer: ```suggestion
	// isAcked is true if Ack has been called on this message.
```

	isAcked bool
}

type AckID interface{}

// Ack acknowledges the message, telling the server that it does not need to
// be sent again to the associated Subscription. This method returns

> // immediately. Calling this method twice on the same message causes a panic.

Reviewer: ```suggestion
// immediately. If Ack has already been called on the message, Ack panics.
```
